services/QuillLMS/spec/support/shared/routing_url_helpers.rb

```diff
@@ -0,0 +1,7 @@
+RSpec.shared_context "routing url helpers" do
```

Comment:
Not required, but since it's logic, you could consider using a constant string for this name that others can use to prevent future typos, e.g. `Rspec::ROUTING_URL_HELPERS`